src/t_list.c

```diff
@@ -58,9 +56,8 @@ static void listTypeTryConvertListpack(robj *o, robj **argv, int start, int end,
         add_length = end - start + 1;
     }
 
-    quicklistSizeAndCountLimit(server.list_max_listpack_size,&sz_limit,&count_limit);
-    if (lpBytes(o->ptr) + add_bytes > sz_limit || lpLength(o->ptr) + add_length > count_limit ||
-        !lpSafeToAdd(o->ptr, add_bytes))
```

Comment:
so we no longer need the `lpSafeToAdd` (listpack concern), because the quicklist concerns are covering this anyway?